### ISSUE TITLE
example、

### ISSUE THREAD

**Issue by bonbonjour** [Sekunde/3D-SIS#14]

**Comment by bonbonjour** [comment]
when I run example.sh, I noticed that testing on whole scan is 0%, is it normal?
this is part of screem shot:
 loaded mode from example_result/scannet_benchmark_checkpoint/ScanNet/example/checkpoints/step_1205541.pth
starting test on whole scan....
  0%|                                                                                                                                                                                                                        | 0/5 [00:00<?, ?it/s]/home/miniconda3/envs/3d-sis-8.0/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:04<00:00,  1.05it/s]
It took 4.797s for test on whole scenes

**Comment by Sekunde** [comment]
from the part you paste, it looks normal. 0 you mean mAP? if so, it is not normal the map is 0. Do you get some visualization, how does it look like?

**Comment by bonbonjour** [comment]
This is the part of inference screen output(running "python main.py --cfg ScanNet/example --step 1205541 --output_dir example_result/scannet_benchmark_checkpoint --mode benchmark --gpu 0" in example.sh).  I guess 0% means it didn't  perform inference on the whole scan, and 100% perform scenes inference, I am not sure whether the  0% is normal.

**Comment by Sekunde** [comment]
you are right. this 0% is just the percentage of the progress bar. if everything is correct in running example.sh, you should be able to find the ply file of visualization in the corresponding folder.

**Comment by bonbonjour** [comment]
when running example.sh, no error occurred. And there are ply files in visualization folder, but I can't open the ply files by cloudcompare, this is part of the ply file content:
ply
format ascii 1.0
element vertex 126879
property float x
property float y
property float z
property uchar red
property uchar green
property uchar blue
element face 0
property list uchar uint vertex_indices
end_header
0.500088 0.494341 0.177517 0 0 0
0.492598 0.490703 0.182717 0 0 0
0.489316 0.489291 0.170277 0 0 0
0.499378 0.500784 0.186239 0 0 0
0.507296 0.488348 0.371781 0 0 0
0.425359 0.503652 0.364962 0 0 0
0.431594 0.501174 0.375987 0 0 0
0.489970 0.482518 0.270122 0 0 0
0.502833 0.484806 0.304741 0 0 0
0.454039 0.497579 0.415815 0 0 0
0.435735 0.497591 0.324599 0 0 0
0.433258 0.497422 0.345172 0 0 0
0.453000 0.500266 0.457000 0 0 0
0.467964 0.495290 0.432702 0 0 0
0.466657 0.497949 0.471185 0 0 0
0.506304 0.481383 0.265646 0 0 0
0.430568 0.505367 0.428415 0 0 0
0.444750 0.502005 0.437500 0 0 0
0.443719 0.501652 0.416013 0 0 0
0.432423 0.504524 0.414534 0 0 0
0.492639 0.495231 0.478461 0 0 0
0.494852 0.492158 0.415004 0 0 0
0.500267 0.492801 0.444554 0 0 0
0.505420 0.478952 0.228830 0 0 0
0.473808 0.480327 0.191195 0 0 0
0.503070 0.480094 0.200326 0 0 0
0.448888 0.492278 0.306646 0 0 0
0.476320 0.489482 0.353837 0 0 0
0.474750 0.487788 0.329500 0 0 0
0.458102 0.492879 0.349369 0 0 0
0.469345 0.492228 0.372761 0 0 0
0.469452 0.493347 0.405750 0 0 0
0.497039 0.491193 0.390617 0 0 0
0.457808 0.489618 0.313509 0 0 0
0.490368 0.498700 0.506255 44370 50745 59160
0.461834 0.501761 0.504106 44370 50745 59160
0.468725 0.487494 0.162468 0 0 0
0.484943 0.487879 0.178130 0 0 0
0.451203 0.484098 0.189170 0 0 0
0.456261 0.484233 0.169277 0 0 0
0.475865 0.484574 0.183310 0 0 0

**Comment by Sekunde** [comment]
can you try meshlab?

**Comment by bonbonjour** [comment]
thanks for your kind help! I can visualize the reault by meshlab
